src/FirstInstall/FirstInstallInstructions.jsx

```diff
@@ -48,20 +47,109 @@ import { hideFirstSteps, selectedVersion } from '../Manager/managerReducer';
+                that you want to work with<br />
+                <Ie /><code>{suggestExampleCmakelists}</code>
+            </li>
+            <li>
+                <b>Board Directory</b> - the location of the board description
+                of the board for which to build the project<br />
+                <Ie /><code>{exampleBoardDir}</code>
+            </li>
+            <li>
+                <b>Board Name</b> - the board name (select from the list that
+                is populated based on the board directory)<br />
+                <Ie /><code>{boardName}</code>
+            </li>
+            <li>
+                <b>Build Directory</b> - the directory in which to run the build
+                (automatically filled based on the board name, but you can
+                specify a different directory)<br />
+                <Ie /><code>{suggestExampleBuildDir}</code>
+            </li>
+            <li>
+                <b>Clean Build Directory</b> - select this option to ensure
+                that you are not building with an outdated build cache
+            </li>
+        </ul>
+    );
+};
+
+const NewSteps = () => (
+    <ul>
+        <li>
+            <b>nRF Connect SDK Release</b> - Select the NCS version that you
+            want to work with.<br />
+            The drop-down list contains the current version of all NCS
+            installation directories that SES knows about.
+        </li>
+        <li>
+            <b>nRF Connect Toolchain Version</b> - Select the version of the
+            toolchain that works with the selected NCS version.<br />
+            The drop-down list contains only toolchain versions that are
+            compatible with the selected NCS version.
+        </li>
+        <li>
+            <b>Projects</b> - Select the project that you want to work with.
+            <br />
+            The drop-down list contains a selection of samples and
+            applications from the sdk-nrf and sdk-zephyr repositories.
+            Select any of the checkboxes underneath to add the samples from
+            that area to the drop-down list. To add projects to the
+            drop-down list, for example, your own custom projects, click
+            {' '}<Dotdotdot /> and select the folder that contains the
+            projects that you want to add.
+        </li>
+        <li>
+            <b>Board Name</b> - Select the board that you want to work with.
+            <br />
+            The drop-down list contains the build targets for all Nordic
+            Semiconductor boards that are defined in the sdk-nrf and
+            sdk-zephyr repositories. Select any of the checkboxes underneath
+            to add the build targets from that area to the drop-down list.
+            To add build targets to the drop-down list, for example, targets
+            for your own custom board, click <Dotdotdot /> and select the
+            folder that contains the board definitions.
```

Comment:
Similar to above:
```suggestion
            Select any of the checkboxes 
            to add the respective build targets to the drop-down list.
            To add custom build targets to the drop-down list, 
            click <Dotdotdot /> and select the
            folder that contains the board definitions.
```

Reply:
Should we add a tip for non-secure applications here? Since the most frequent issue for new users is that they don't select the *ns target?
Something like:
`Note that if you work with a TrustZone device, you usually build the firmware for the non-secure domain and must therefore select the non-secure build target, for example, ``nrf9160dk_nrf9160ns`` or  ``nrf5340pdk_nrf5340_cpuappns``.`

Reply:
I added the suggested note.